corehq/tabs/tabclasses.py

```diff
@@ -827,6 +827,16 @@ def sidebar_items(self):
                     'subpages': subpages
                 })
 
+            if toggles.SUPERSET_ANALYTICS.enabled(self.domain):
+                from corehq.apps.export.views.list import CommCareAnalyticsListView
+                export_data_views.append({
+                    'title': _(CommCareAnalyticsListView.page_title),
```

Comment:
no need to translate - `page_title` is already translated, and translating variables doesn't work anyway